sdk/src/main/scala/io/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -260,12 +271,17 @@ class AccountForgeMessageBuilder(
               require(ommers.isEmpty, "No Ommers allowed for the last block of the withdrawal epoch.")
 
               val withdrawalEpochNumber: Int = WithdrawalEpochUtils.getWithdrawalEpochInfo(mainchainBlockReferencesData.size, dummyView.getWithdrawalEpochInfo, params).epoch
-              val distributionCap = if (Version1_4_0Fork.get(consensusEpochNumber).active) {
+              val distributionCap = if (fork_1_4_active) {
                 val mcLastBlockHeight = params.mainchainCreationBlockHeight + ((withdrawalEpochNumber + 1) * params.withdrawalEpochLength) - 1
```

Comment:
In AccountState, mcLastBlockHeight is calculated without adding 1 to whitdrawalEpoch, i.e.:
  `      val mcLastBlockHeight = params.mainchainCreationBlockHeight + (modWithdrawalEpochInfo.epoch * params.withdrawalEpochLength) - 1`
Is it correct?